src/LiveComponent/src/Util/LiveControllerAttributesCreator.php

```diff
@@ -98,6 +98,13 @@ public function attributesForRendering(MountedComponent $mounted, ComponentMetad
             $mountedAttributes = $mountedAttributes->defaults(['data-live-id' => $id]);
         }
 
+        $requestMethod = AsLiveComponent::getMethod($mounted->getComponent());
+
+        if (!isset($mountedAttributes->all()['data-live-request-method-value'])) {
```

Comment:
I think it's safe to set this in all situations - without the `if` statement.

Reply:
Done